webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/GeneralScrambleSheet.kt

```diff
@@ -44,18 +43,18 @@ class GeneralScrambleSheet(scrambleRequest: ScrambleRequest, globalTitle: String
 
         val scrambleImageSize = scrambleRequest.scrambler.getPreferredSize(maxScrambleImageWidth, maxScrambleImageHeight)
 
-        // First check if any scramble requires highlighting.
-        val forceStdHighlighting = requiresHighlighting(availableWidth, scrambleImageSize, scrambleRequest.scrambles, STD_SCRAMBLE_PREFIX)
+        val indexColumnWidth = availableWidth / 20f
```

Comment:
Where does that 20 come from?
I think it would make sense to (a) briefly document it and (b) move it into the companion object as a `const val`